source/conf.py

```diff
@@ -200,8 +200,9 @@ def setup(app):
   app.connect('env-before-read-docs', build_partials)
   app.connect('source-read', source_handler)
   app.connect('html-page-context', update_body)
-  app.add_javascript('js/custom.js')
-  app.add_javascript('https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js')
+  app.add_js_file('js/custom.js')
+  app.add_js_file('https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js')
+  app.add_js_file('https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js')
```

Comment:
You might also need to change the `app.add_stylesheet('css/custom.css')` function in line 210 to `app.add_css_file('css/custom.css')`.